### ISSUE TITLE
Fresh install

### ISSUE THREAD

**Issue by longsack** [jrudess/streamdvr#74]
Fresh install, have an issue:
MFCSL    [ERROR] MissMolly error: No plugin can handle URL: https://www.myfreecams.com/#MissMolly

I get "No plugin can handle URL:" for a lot of the plugin's.

I have checked /root/.config/streamlink/plugins the directory is not at this location.
Seems like streamlink is installed: 
streamlink -v
[cli][info] streamlink is running as root! Be careful!
usage: streamlink [OPTIONS] <URL> [STREAM]

I think its a plugin location issue in one of the conf files, just not sure what one.... or if I have another issue.

Thoughts?

**Comment by longsack** [comment]
More info... 
find / -name "streamlink"
/usr/local/bin/streamlink
/usr/local/lib/python3.4/dist-packages/streamlink

I tried loading the "plugins" folder from the above into the /root/.config/streamlink (and also plugins).

still get the same error

**Comment by Asynchr0nous** [comment]
if you want to use streamlink for MFC you need the "hlssession" plugin which is NOT inlcuded in normal streamlink version

**Comment by longsack** [comment]
i'm learning that.  and see the plugins link is a 404, not sure what to do at this point or how to move forward.

**Comment by longsack** [comment]
I tried loading the "plugins" folder from the above into the /root/.config/streamlink (and also plugins).

Sorry, what I mean is I tried copy/paste the plugins into the /root/.config/streamlink folder without any luck.  I can get a copy of the "hlssession" plugin, but not sure of the location to put it, in the /root/.config/stramlink", folder?  or maybe the python3.4/dis-packages/streamlink folder above?

Streamlink seems to be installed correctly as it will play/download videos alone.  However when combined with StreamDVR it seems I get the "No plugin can handle URL:"... So thinking it's maybe a directory issue in streamdvr perhaps?

**Comment by Asynchr0nous** [comment]
solution is in the comments: https://github.com/jrudess/streamdvr/issues/27

**Comment by longsack** [comment]
Got this to work... I had an old install on another system. I copied the plugins folder to a few different locations (along with the hlssessions plugin).
the locations above are where I copied them to.  I'm not sure what one specifically as I copied to:
/usr/local/bin/streamlink
/usr/local/lib/python3.4/dist-packages/streamlink
/root/.config/streamlink 

but one of them was the right one... :)  Now now issue.!
